lib/plivo/resources/calls.rb

```diff
@@ -199,7 +200,7 @@ def send_digits(digits, leg = nil)
 
       def cancel_request
         resource_path = @_resource_uri.sub('Call', 'Request')
-        @_client.send_request(resource_path, 'DELETE', nil)
+        @_client.send_request(resource_path, 'DELETE', nil, nil, false , is_voice_request: @_is_voice_request)
```

Comment:
Please make sure other teams' api requests are not affected by this change.

Reply:
Default params are being sent since some params haven't been mentioned while making function call